src/couchapps/WorkQueue/views/elementsDetailByWorkflowAndStatus/reduce.js

```diff
@@ -1,17 +0,0 @@
-function (key, values, rereduce) {
```

Comment:
when reduce function doesn't exist, default is reduce=false. But I think it is better to have explicit since if we later add reduce function we don't have to change this again.